kitsune/sumo/views.py

```diff
@@ -26,6 +27,12 @@ def locales(request):
     return render(request, template, dict(next_url=get_next_url(request) or reverse("home")))
 
 
+@never_cache
+@require_GET
+def csrftoken(request):
+    return render(request, "sumo/csrftoken_for_form.html")
```

Comment:
no need to render the an html page here. We could use the `django.middleware.csrf.get_token()` here

```python
from django.middleware.csrf import get_token()
from django.http import JsonResponse

def csrf_token(request):
   return JsonResponse('csrfToken': get_token(request)
```

and use `fetch` directly on the url

Reply:
I think we still need to use Django's `csrf_token` template tag here because it returns HTML meant to be embedded in a form, a hidden input element with the CSRF token, which is what we then replace the placeholders with:
```html
<input type="hidden" name="csrfmiddlewaretoken" value="7T1AMhLMl9SKaKnp6u0vIK5Pb3wEtMxVKzMgrRE5c5EjjaxZb2otyuEnp7Ti45RL">
```

Reply:
Actually, I'll do what you suggested, in that I'll get the token, and simply replace the `value` attribute of any hidden `input` elements with `name="csrfmiddlewartoken"` within all forms.